www/README.md

```diff
@@ -21,3 +30,14 @@ module.exports = {
   ],
 };
 ```
+
+The second line should ideally include a description of the example
+```js
+// snowpack.config.js
+// Example: enabling the sass plugin after installing it `npm install --save-dev @snowpack/plugin-sass`
```

Comment:
Nit, based on our convo, I think we agreed on:

```
// snowpack.config.js
// Example: Connect the Sass plugin
// [npm install @snowpack/plugin-sass]
```

Where:
- 1st line: required if this a code snippet about a specific file
- 2nd line: describe the change, not always required but recommended. Start with `Example:` if relevent.
- 3rd line: if a dependency is implied, add the npm install script here